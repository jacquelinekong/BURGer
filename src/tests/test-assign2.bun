/*string butterfly;
butterfly = "monarch";
bool pretty;
pretty = true;

print(butterfly);
println(" butterflies are pretty if this prints 1: ");
println(pretty);*/

string butterfly = "monarch";
bool pretty = true;
print(butterfly);
println(" butterflies are pretty if this prints 1: ");
println(pretty);
