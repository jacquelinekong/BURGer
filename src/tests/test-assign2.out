monarch butterflies are pretty if this prints 1:
1
